src/components/Register.js

```diff
@@ -1,24 +1,44 @@
 import React, { useState } from 'react'
 import { register } from '../auth'
 import { useNavigate } from 'react-router-dom'
+import { ToastContainer, toast } from 'react-toastify'
+import 'react-toastify/dist/ReactToastify.css'
 
 const Register = ({ setShowRegister }) => {
   const [email, setEmail] = useState('')
   const [password, setPassword] = useState('')
 
+  // Navigation
   const navigate = useNavigate()
 
+  // Register popup
+  const popUp = () => {
+    setShowRegister(false)
+  }
+
+  // Toastify error
+  const errorMessage = () => {
+    toast.error('Email or password are not correctly written, please try again', {
+      position: 'top-center',
+      autoClose: 5000,
+      hideProgressBar: false,
+      closeOnClick: true,
+      pauseOnHover: true,
+      draggable: true,
+      progress: undefined
+    })
```

Comment:
Configure this in the toast container.
https://fkhadra.github.io/react-toastify/introduction/